app/src/main/java/com/github/braillesystems/learnbraille/res/GolubinaCourse.kt

```diff
@@ -230,9 +230,14 @@ internal val golubinaIntroLessons by lessons {
                     Они получаются из букв А, Б, Ц, Д, Е добавлением цифрового знака.
                     Например, цифра 3, как и число 3 - это цифровой знак + Ц.
                     Число двадцать четыре - это цифровой знак, затем буквы Б и Д. 
-                    В уроках мы для краткости не будем всякий раз ставить цифровой знак."""
+                    В следующих шагах просмотрите, а затем введите число «двенадцать тысяч
+                    триста сорок пять»."""
         )
-        showAndInputChars("12345")
+        +Show(content.markers.getValue(MarkerType.NumberSign))
```

Comment:
It is more consistent to abstract it away too.

Reply:
Well, it's not the only place where a "bare" number sign is left in the course. So I suppose it is no worse than any other ShowMarker step.

Reply:
But I've added this to #257